war/src/main/js/add-item.js

```diff
@@ -1,12 +1,12 @@
 // Initialize all modules by requiring them. Also makes sure they get bundled (see gulpfile.js).
 var $ = require('jquery-detached').getJQuery();
 
-var getItems = function(){
+var getItems = function() {
   var d = $.Deferred();
   $.get('itemCategories?depth=3').done(
-      function(data){
-        d.resolve(data);
-      }
+    function(data){
+      d.resolve(data);
+    }
```

Comment:
@batmat No, I took advantage this revision of _New Item_ form in order to have the same coding style convention in this file.